src/labels/CustomLabels.labels

```diff
@@ -1624,6 +1632,22 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Validate Recurring Donations data if it is valid install period</shortDescription>
         <value>{0} is not a supported value for Installment Period. Select Daily, Weekly, 1st and 15th, Monthly, or Yearly.</value>
     </labels>
+    <labels>
+        <fullName>RD2_PageNotAvailableHeader</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>RD2_PageNotAvailableHeader</shortDescription>
+        <value>Page Not Available</value>
+    </labels>
+    <labels>
+        <fullName>RD2_PageNotAvailableMessage</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>RD2_PageNotAvailableMessage</shortDescription>
+        <value>Data is not available when Enhanced Recurring Donations are enabled</value>
```

Comment:
@sam-knox , can you please advise on test to be displayed on Custom Installment and Field Mapping pages when enhanced RDs are enabled but menu option is still there until NPSP Settings page refresh?

Reply:
@RadSFDeveloper - Would the user simply need to refresh their browser, or would they need to click on NPSP Settings again to re-load it and clear out the two disabled settings pages?

Reply:
Just refresh the NPSP Settings page if/when they need.

Reply:
Ok, how about this text for RD2_PageNotAvailableMessage:

"This page is not available in Enhanced Recurring Donations. Refresh your browser and return to NPSP Settings."

Reply:
Thank you @sam-knox !